docs/pathways/ontology-engineer.md

```diff
@@ -29,7 +29,6 @@ As a ontology engineer, it would be useful for you to know how curators work, as
 1. [Setting up a ODK repository](../tutorial/setting-up-project-odk.md)
 1. [Developing an obo ontology](../lesson/developing-an-obo-ontology.md)
 1. [Understanding product variants](../explanation/owl-format-variants.md)
-1. [Dealing with large ontologies](../howto/deal-with-large-ontologies.md)
```

Comment:
Removed this from "learning..." section because it was in the "how to..." section.